.travis.yml

```diff
@@ -1,7 +1,18 @@
+language: java
```

Comment:
So you need to use something like this https://github.com/Gregam3/ASEGroup1/blob/56b1e834e4ff0ebd2fa909d31248453990e8afb2/.travis.yml#L1 for multiple builds I assume this is why the build is failing If you want I can merge in my changes to avoid conflicts